tmux has a stack overflow in CSI parsing.

OpenBSD 6.8 errata 003, October 29, 2020:

https://ftp.openbsd.org/pub/OpenBSD/patches/6.8/common/003_tmux.patch.sig

diff -u -p -u -r1.183 input.c
--- tmux/input.c	19 Aug 2020 06:37:23 -0000
+++ tmux/input.c	29 Oct 2020 15:34:14 -0000
@@ -1976,8 +1976,13 @@ input_csi_dispatch_sgr_colon(struct inpu
 				free(copy);
 				return;
 			}
-		} else
+		} else {
 			n++;
+			if (n == nitems(p)) {
+				free(copy);
+				return;
+			}
+		}
 		log_debug("%s: %u = %d", __func__, n - 1, p[n - 1]);
 	}
 	free(copy);
